### ISSUE TITLE
Fails when a non-matching expected value is a string containing backslashes

### ISSUE THREAD

**Issue by kerryb** [assert-value/assert_value_elixir#12]
assert_value version: 0.9.5
elixir version: 1.13.3
otp version: 24.2.1
OS: macOS 12.2.1 (21D62)

Test case:

```elixir
defmodule FooTest do
  use ExUnit.Case
  import AssertValue

  test "handles escapes in non-matching expected string" do
    actual = ~s["foo"]
    assert_value actual == "\"bar\""
  end
end
```

Output:

```
09:08:50.354 [error] GenServer AssertValue.Server terminating
** (MatchError) no match of right hand side value: []
    (elixir 1.13.3) src/elixir_interpolation.erl:74: :elixir_interpolation.extract_char/8
    (elixir 1.13.3) src/elixir_tokenizer.erl:755: :elixir_tokenizer.handle_strings/6
    (elixir 1.13.3) src/elixir.erl:357: :elixir.string_to_tokens/5
    (elixir 1.13.3) lib/code.ex:903: Code.string_to_quoted/2
    (assert_value 0.9.5) lib/assert_value/parser.ex:160: AssertValue.Parser.code_match_ast?/2
    (assert_value 0.9.5) lib/assert_value/parser.ex:143: AssertValue.Parser.find_ast_in_code/3
    (assert_value 0.9.5) lib/assert_value/parser.ex:69: AssertValue.Parser.parse_assert_value/5
    (assert_value 0.9.5) lib/assert_value/server.ex:139: AssertValue.Server.prepare_formatted_diff_and_new_code/2
    (assert_value 0.9.5) lib/assert_value/server.ex:76: AssertValue.Server.handle_call/3
    (stdlib 3.17) gen_server.erl:721: :gen_server.try_handle_call/4
    (stdlib 3.17) gen_server.erl:750: :gen_server.handle_msg/6
    (stdlib 3.17) proc_lib.erl:226: :proc_lib.init_p_do_apply/3
```

This tends to crop up for example when checking chunks of generated HTML, where quotes around attribute values get escaped. It’s fine if the expected value is written as `~s["bar"]`, but this isn’t the form that _assert_value_ generates if run with no expected value.

**Comment by smetana** [comment]
Noted. Thanks. I cannot help with this immediately. I am in Kyiv right now. Will get back to this after Kyiv will be secured from russian troops and air raids. PR is welcomed.

**Comment by kerryb** [comment]
Thanks for the reply – suddenly a minor bug in a test library seems very unimportant. I’ll have a go at fixing it and submit a PR if I’m successful.

My heart goes out to you and everyone else in Kyiv and the rest of Ukraine. Stay safe, and I wish your country a speedy victory. 🇺🇦

**Comment by smetana** [comment]
Hi. Sorry for delay

Fixed in 91cfeea. You can use it as ::ref dependency
```elixir
{:assert_value, git: "https://github.com/assert-value/assert_value_elixir.git", ref: "91cfeeab90e73efa73a761cf18e614c0b81c7d32"}
```
or wait a bit more for the next release.

**Comment by smetana** [comment]
Released 0.10.0
